hazelcast/src/main/java/com/hazelcast/partition/impl/InternalPartitionServiceImpl.java

```diff
@@ -1530,6 +1540,23 @@ public void onPartitionLost(InternalPartitionLostEvent event) {
         eventService.publishEvent(SERVICE_NAME, registrations, partitionLostEvent, event.getPartitionId());
     }
 
+    public List<ReplicaSyncInfo> getOngoingReplicaSyncRequests() {
```

Comment:
Can you add documentation to these methods? We need to make sure we don't add more technical debt to these objects.